longshot/application/main.tf

```diff
@@ -61,10 +61,11 @@ resource "heroku_app" "app" {
 
   config_vars {
     # App settings:
-    APP_ENV   = "production"
-    APP_DEBUG = "false"
-    APP_LOG   = "errorlog"
-    APP_URL   = "https://${var.host}"
+    APP_ENV                    = "production"
+    APP_DEBUG                  = "false"
+    APP_LOG                    = "errorlog"
+    APP_URL                    = "https://${var.host}"
+    TRUSTED_PROXY_IP_ADDRESSES = "**"
```

Comment:
does this mean we trust everything?

Reply:
Yup, since we don't know the IP addresses for Heroku's load balancer. If we were hosting this behind a service with a known IP address (like we used to do in AWS), we could have set that here instead.

Reply:
And if so, would having the HA Proxy IP address be listed here be a better solution? Or is this a requirement for HTTPS to work properly at all?